src/main/java/edu/kis/powp/jobs2d/lines/CustomLine.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.powp.jobs2d.lines;
+
+import edu.kis.legacy.drawer.shape.line.BasicLine;
+
+import java.awt.*;
+
+public class CustomLine extends BasicLine {
```

Comment:
There is AbstractLine class that would be better than BasicLine (because CustomLine is not a BasicLine)